pkg/model/record.go

```diff
@@ -92,7 +92,7 @@ func NewRecord(
 
 func Accumulate(r *ebpf.BpfFlowMetrics, src *ebpf.BpfFlowMetrics) {
 	// time == 0 if the value has not been yet set
-	if r.StartMonoTimeTs == 0 || r.StartMonoTimeTs > src.StartMonoTimeTs {
+	if r.StartMonoTimeTs == 0 || (r.StartMonoTimeTs > src.StartMonoTimeTs && src.StartMonoTimeTs != 0) {
```

Comment:
isn't the `&&` logic is redundant ?

Reply:
no, first is "r.StartMonoTimeTs", second is "src.StartMonoTimeTs"